lib/three_scale/domain_substitution.rb

```diff
@@ -149,9 +149,6 @@ def match_internal_admin_domain?(host)
     def match_internal_domain?(host)
       internal_domain == host
     end
-
-    deprecate domain: "use #internal_domain or #external_domain",
```

Comment:
same here. If we don't remove the method or make it private, deprecation should stay.